DMX.Core.Api.Tests.Acceptance/APIs/Labs/LabApiTests.Logic.cs

```diff
@@ -0,0 +1,38 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DMX.Core.Api.Tests.Acceptance.Models.Labs;
+using FluentAssertions;
+using Force.DeepCloner;
+using Xunit;
+
+namespace DMX.Core.Api.Tests.Acceptance.APIs.Labs
+{
+    public partial class LabApiTests
+    {
+        [Fact]
+        public async Task ShouldAddLabAsync()
+        {
+            // given
+            Lab randomLab = CreateRandomLab();
+
```

Comment:
Can we remove the empty lines here? Or does it need to be separated because of the assignment?